core/src/main/java/nl/nn/adapterframework/pipes/CsvParserPipe.java

```diff
@@ -119,12 +124,9 @@ public void setFieldSeparator(String fieldSeparator) {
 		this.fieldSeparator = fieldSeparator;
 	}
 
-	@IbisDoc({"4", "When set, character casing will be changed for the header. Possible values ['lowercase', 'uppercase']","not set"})
-	public void setHeaderCase(String headerCase) {
-		this.headerCase = EnumUtils.parse(HeaderCase.class, headerCase);
-	}
-	public HeaderCase getHeaderCaseEnum() {
-		return headerCase;
+	@IbisDoc({"4", "When set, character casing will be changed for the header.","not set"})
```

Comment:
ik denk dat het beter is `,"not set"` weg te halen, dan wordt hij als NULL gezien en is dan dus 'not set'.